### ISSUE TITLE
Very strange result on predict/evaluate

### ISSUE THREAD

**Issue by Romiroz** [apple/tensorflow_macos#192]
Good day,

Strange results with Conv1D or LSTM layers. Good results on training, with same data on evaluate accuracy is above zero. 
![image](https://user-images.githubusercontent.com/66079444/110231594-b317cc00-7f29-11eb-8ab1-1a52a6d711d0.png)
But on the PC wit TF 2.4 with this notebook all OK. And strange - if i press 'Stop' in jupyter during training on Mac, evaluate accuracy will be normal.

**Comment by Riccorl** [comment]
Yes, it is a known bug (#145). I think it will be fixed eventually.
